kubetest/azure.go

```diff
@@ -424,9 +424,30 @@ func newAKSEngine() (*Cluster, error) {
 		return nil, err
 	}
 
+	if err := c.installAzureCLI(); err != nil {
+		return nil, err
+	}
+
 	return &c, nil
 }
 
+func (c *Cluster) installAzureCLI() error {
+	if err := control.FinishRunning(exec.Command("pip", "install", "azure-cli")); err != nil {
```

Comment:
could we check whether "az" exist before install? we shouldn't override existing Azure CLI if it has already been installed.